database/seeds/WikisSeeder.php

```diff
@@ -1,44 +1,89 @@
 <?php
 
+use App\QueryserviceNamespace;
+use App\User;
 use App\Wiki;
+use App\WikiDb;
 use App\WikiDomain;
 use App\WikiManager;
 use App\WikiSetting;
+use App\WikiSiteStats;
 use Illuminate\Database\Seeder;
-use Illuminate\Support\Facades\DB;
+use Illuminate\Support\Facades\Config;
+use Illuminate\Support\Facades\Hash;
 use Illuminate\Support\Str;
 
 class WikisSeeder extends Seeder
 {
-    public function run()
+    /**
+     * @param \App\User $user
+     * @param string $name
+     * @param array $stats
+     */
+    public function createWiki($user, $name, $stats)
```

Comment:
Would this be better as private? Are you expecting to call it anywhere other than `$this->run()`?